Tools/patchcheck/patchcheck.py

```diff
@@ -77,7 +77,7 @@ def get_git_remote_default_branch(remote_name):
 
     It is typically called 'main', but may differ
     """
-    cmd = "git remote show {}".format(remote_name).split()
+    cmd = f"git remote show {remote_name}".split()
```

Comment:
Remove the split call?:

```suggestion
    cmd = ("git", "remote", "show", remote_name)
```

Reply:
Let's keep the original, it's more greppable for the actual command.